_data/whats-new.yml

```diff
@@ -4,8 +4,71 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Aug 26 22:12:19 2019
+updated: Mon Sep 9 16:43:06 2019
 entries:
+- description: Added an example of the `escapeCss` function to the [Templates XSS
+    security](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html)
+    topic.
+  versions: 2.3.x
+  type: Major update
+  date: September 6, 2019
+  link: https://github.com/magento/devdocs/pull/5299
+- description: Added more information about the `output` attribute for containers
+    in the [Layout Instructions topic](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: September 6, 2019
+  link: https://github.com/magento/devdocs/pull/5332
+- description: 'Updated the Cloud Release notes for [ece-tools version 2002.0.21](https://devdocs.magento.com/guides/v2.3/cloud/release-notes/cloud-tools.html#v2002021):<br/>-
+    Added environment variables for Cloud deployment: [CONSUMERS_WAIT_FOR_MAX_MESSAGES](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#consumers_wait_for_max_messages)
+    and [LOCK_PROVIDER](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#lock_provider)<br/>-
+    Added a [service compatibility table](https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services.html)
+    that shows support for installed software and service versions such as Elasticsearch,  RabbitMq,
+    Redis, and DB on each Magento version.<br/>- Updated the [Launch Docker](https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-config.html)
+    procedure to remove references to the deprecated `db:docker:convert` command and
+    Docker ENV files which are no longer generated during the Docker build.<br/>-
+    Added [Docker images](https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-development.html)
+    to support PHP 7.3 and Varnish Cache 6.2.<br/>- Updated the Docker PHP images
+    to add Node.js to support node, npm, and the grunt-cli capabilities inside the
+    [PHP container](https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-development.html#php-fpm-container).'
+  versions: 2.x
+  type: Major update
+  date: September 5, 2019
+  link: https://github.com/magento/devdocs/pull/5279
+- description: Added the [Reference a CMS block](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html#ref_cms_block)
+    section to the 'Common Layout Customization Tasks' topic.
```

Comment:
The topic name doesn't need to be in quotes does it?

Reply:
Well APA Style says: 

Use double quotation marks for title of an article, a chapter, or a web page. Use italics for title of a periodical, a book, a brochure or a report.

We have used both single and double quotes, but if it's a topic it should be " ".  So moving forward, that's the rule.

Reply:
I don't recall us ever using quotation marks for the title of an article, chapter, or web page. Maybe I'm wrong. 

If this isn't already in the style guide in the wiki, please add it.

Reply:
In fact, using quotes around topic titles looks/seems strange. 🤷‍♀

Reply:
Our formatting style guide rule:

> References - print
> Use italics to denote the title of a book or other publication intended for recommendation or references use.
> 
> Example: _Merriam Webster's Collegiate Dictionary_ or _Moby Dick_


Any changes of style have to be approved by the Docs team.

Reply:
Yes, I know that is they style for hardprinted things, like for the titles of books. What gave me pause was the "article" or "web page" portion of that. Article denotes "on the web"---which seems to be a strange use case for quotation marks.

Just my two cents.

Reply:
In the What's new, we have topics encased in both single and double quotes.  I was wondering why we were not using italic formatting for topic titles, which is what I've used in the past.